zerver/lib/event_schema.py

```diff
@@ -174,6 +174,8 @@
     ],
     optional_keys=[
         ("display_in_profile_summary", bool),
+        ("rendered_name", str),
```

Comment:
These fields are always included, so this should be in the `required_keys` with type set to `OptionalType(str)` to handle the case where it can be `None`.